data/registry/instrumentation-go-ibm-sarama.yml

```diff
@@ -0,0 +1,16 @@
+# cSpell:ignore sarama
+title: IBM/sarama Instrumentation
+registryType: instrumentation
+language: go
+tags:
+  - sarama
+  - instrumentation
+  - go
+license: Apache 2.0
+description: Package sarama provides functions to trace the IBM/sarama package.
```

Comment:
```suggestion
description: Package sarama provides functions to trace the [IBM/sarama](https://github.com/IBM/sarama) package.
```

Reply:
Must `sarama` be lowercase and non-monospaced?

Seems to be capitalized here: https://github.com/IBM/sarama in some instances.

Reply:
It is better capitalized  as Go imports are case-sensitive.